### PR TITLE
Function for to clean the CNPJ

### PR DESCRIPTION
**Pull request by jeremiassantos**
Function for to clean the CNPJ

### CODE REVIEW

**Comment by jeremiassantos** [comment]
test

**Review by erickamarante — APPROVED** [review]

File: index.js [hunk 1]
```diff
@@ -16,12 +16,27 @@ const digit = numbers => {
...
 /**
  * Validates a CNPJ
  * @param {String|Number} cnpj The CNPJ value to be validated
  * @return {Boolean}
  */
 export function validate(cnpj) {
+  // Verify reference
+  if(!cnpj) {
+    return false
+  }
```
Comment (gabeins) [inline index.js:39]: It's unnecessary to check this here, since it's being handled by line 45. Please remove.

File: test.js [hunk 2]
```diff
@@ -37,15 +37,27 @@ test('invalid sequential CNPJs', t => {
   t.is(validate(88888888888888), false)
   t.is(validate(99999999999999), false)
   t.is(validate('00000000000000'), false)
+  t.is(validate(null), false)
```
Comment (gabeins) [inline test.js:40]: I understand the reasoning for this test, but it should be in the block for "invalid unformatted CNPJs" and not on the "invalid sequential CNPJs".

Honestly I should have added this check before, thanks for catching it. I would like this to be on a separate PR, since it's not related to adding the striping of non digit characters as you propose.

**Review by gabeins — CHANGES_REQUESTED** [review]
Hi @jeremiassantos! 

First, let me apologise for the long time reviewing this. Somehow I had unwatched my own repository and didn't get notified of this PR. Sorry.

I have 2 points about this PR:

1. We are using `yarn` instead of `npm`, please use that and remove `package-lock.json`;
2. Thinking about this feature, `cleanse` is not a good fit, I believe it should be part of the `format()` function, probably with an options object as a second parameter with `{ strip: true }`. What do you think?

**Comment by gabeins** [comment]
Hi @jeremiassantos, are you still interested in moving this PR forward?
